packages/react/src/components/carbon-components-react/UIShell/SideNavMenu.js

```diff
@@ -108,9 +108,21 @@ export class SideNavMenu extends React.Component {
     };
   }
 
+  scrollToTop = () => {
+    const navItemsElem = root.document.getElementsByClassName(
+      'bx--side-nav__items'
+    );
```

Comment:
It'll be nicer if we can better scope the selection of elements (e.g. only the menu items associated with this menu, or go up to the ancestor `.bx--side-nav` and start from there), but going as-is looks OK for now.

Reply:
hmm there's actually only one `bx--side-nav__items` element from `SideNavItems`, it wraps the entire menu. Would it be better to create a ref for the `SideNavMenu` and get the `bx--side-nav__items` from the ref instead?

Reply:
It's certainly the "React" way of doing it...

Reply:
changed to target the `side-nav__items` element from the ref instead